scripts/populate_institutions.py

```diff
@@ -1603,19 +1653,10 @@ def main(env):
                 'email_domains': [],
                 'delegation_protocol': 'saml-shib',
             },
-        ]
-
-    init_app(routes=False)
-    with transaction.atomic():
-        for inst_data in INSTITUTIONS:
-            update_or_create(inst_data)
-        for extra_inst in Institution.objects.exclude(_id__in=[x['_id'] for x in INSTITUTIONS]):
-            logger.warn('Extra Institution : {} - {}'.format(extra_inst._id, extra_inst.name))
+        ],
+}
 
 
 if __name__ == '__main__':
-    env = str(sys.argv[1]).lower() if len(sys.argv) == 2 else None
-    if env not in ENVS:
-        print('An environment must be specified : {}', ENVS)
-        sys.exit(1)
-    main(env)
+
+    main()
```

Comment:
Anything else we should do here?

Reply:
We tend to `init_app` just before calling `main`, but as long as you're not doing Django things between here and there, it should be fine.

Reply:
I will take a look at moving `init_app()` out of and before the main().

Reply:
Done.

`init_app()` was called right before the `with transaction.atomic():` block inside (and at the very end of) `main()` ... Now it has been moved out and put right before `main()`.

```python
if __name__ == '__main__':
    init_app(routes=False)
    main(default_args=False)
```